data/coalition/coalition culture conversations.txt

```diff
@@ -265,6 +284,46 @@ mission "Coalition Departing Trouble - B"
 		conversation
 			`Once you're a good distance from <origin>, your sensors alert you to danger in the system: several explosions and a torrent of projectiles are flying nearby. You swiftly ready yourself and prepare for the threat, only to find Coalition civilian ships blowing up near the middle of the system. Some Heliarch warships are in their midst, moving from one to the other, unbothered by the ship debris as vessels explode around them.`
 			`	They appear to be boarding the few ships that weren't damaged enough to have their hulls ruptured. After a few short minutes, they seem to have seized control of the vessels, and are having them follow the warships out of the system.`
+			branch heliarch
+				has "joined the heliarchs"
+			branch lunarium
+				has "joined the lunarium"
+			branch neither
+				not "joined the heliarchs"
+				not "joined the lunarium"
+			label heliarch
+			`	One of the ships hails you briefly, apparently in a cheery mood as they celebrate having captured the criminals. They wish you a good day and good luck on your duties, and cut the comms as they prepare to leave.`
+				decline
+			label lunarium
+			`	As you glance at the wreckage of the civilian ships, you feel uneasy. Worse still is the thought that hits you soon after, of whether any of the Lunarium leaders you met were aboard any of the ships.`
+				decline
+			label neither
+				decline
```

Comment:
Does this work?

Reply:
It seemed to end it properly when I tested it, yeah.